Automattic/wp-calypso
# My Home: Add new eCommerce cards to Learn & Grow section

#### Changes proposed in this Pull Request

* Add new cards under `Learn & Grow` section for the user under the eCommerce plan
   * 10 ways to find success with your website
   * How to respond to customer feedback

![image](https://user-images.githubusercontent.com/13596067/127599688-c4b62c4c-b02b-48dc-9142-41eb4ce2831b.png)
![image](https://user-images.githubusercontent.com/13596067/127599759-533909ab-395e-4cb3-83b6-94aca701beac.png)

https://user-images.githubusercontent.com/13596067/127460237-c5151ddb-ce35-46e7-86a5-19188a74b228.mov

#### Testing instructions



* Apply D64875-code
* Go to My Home
* Switch cards in `Learn & Grow` section
   * You have to see the new cards if the site is under the eCommerce plan
   * You should not see the new cards if the site is under the eCommerce plan



Close https://github.com/Automattic/wp-calypso/issues/54863
